packages/common/components/elements/company-logo.marko

```diff
@@ -0,0 +1,20 @@
+import { get, getAsObject } from "@base-cms/object-path";
+import { buildImgixUrl } from "@base-cms/image";
+
+$ const { content } = input;
+$ const image = getAsObject(content, "primaryImage");
+$ const blockName = "company__logo";
```

Comment:
The block name shouldn't have double underscores. Double underscores signifies a block _element_. As it is now, the image element is going to be `company__logo__image`, which is "invalid." You'll want the block name to be `company-logo` -- then the element will be `company-logo__image`